src/endpoints/authentication/CMakeLists.txt

```diff
@@ -25,6 +25,7 @@ target_include_directories(
   ${IRODS_EXTERNALS_FULLPATH_FMT}/include
   ${IRODS_EXTERNALS_FULLPATH_SPDLOG}/include
   CURL::libcurl
+  ${jwt_SOURCE_DIR}/include
```

Comment:
Just tried `jwt` and `jwt::jwt`, doesn't seem to work... Renaming it to `jwt-cpp`, matching the name in the project's CMakeLists, allows us to do just that though.

Reply:
What do you mean by **renaming**? What is **it**?

@SwooshyCueb Is there a simple way to determine what name should be used as the import target? (e.g. like `CURL::libcurl)`

Is it by convention? Are there rules for identifying this information?

Reply:
To clarify, changing the `FetchContent_Declare(...)`'s `jwt` to `jwt-cpp` allows use to use `jwt-cpp` instead of `${jwt_SOURCE_DIR}`. Can include in PR but I think the plan is to leave CMake alone until the external item is done.

Reply:
> Can include in PR but I think the plan is to leave CMake alone until the external item is done.

That seems reasonable.   The intent is clear enough for now, and the external PR should be quick.

Reply:
> @SwooshyCueb Is there a simple way to determine what name should be used as the import target? (e.g. like `CURL::libcurl)`
> 
> Is it by convention? Are there rules for identifying this information?

Import target names are usually defined by their find modules or cmake config files. The convention is `project::target` or `project::project`.